subsys/nrf_security/src/drivers/cracen/cracenpsa/src/kmu.c

```diff
@@ -19,6 +19,8 @@
 #include "common.h"
 #include "kmu.h"
 
+#define PROVISIONING_SLOT 250
```

Comment:
Short description here for the process would be nice. Simple things like:
"We reserve one KMU slot to verify that the provisioning process was not interrupted by a power failure or similar reasons. 
We do this by writing the following metadata to the provisioning slot before we store the provisioning our own keys. We only use the provisioning slot metadata for this. 
The provisioning slot metadata are 32 bit long and they are used as follows:
Reserved for future use (16 bits) | Slot id (8 bits) | Number of slots (8 bits)
"